apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/open_trades/trade_state/states/SellerState3a.java

```diff
@@ -250,6 +270,18 @@ public static class View extends BaseState.View<Model, Controller> {
         private View(Model model, Controller controller) {
             super(model, controller);
 
+            txIdValidator = new BitcoinDataValidator() {
```

Comment:
I think this just pollutes the code here. 
`private final BitcoinTxValidator bitcoinTxValidator = new BitcoinTxValidator();`
would be much more concise.

Of course such things are not hard science and diff. ppl will have diff. opinions... just posting mine. Not a showstopper for ACK...

Reply:
@djing-chan thanks for the review. I agree that this last fine tuning just depends on devs preferences.
The `pollution` you are talking about is quite used in the current Bisq2 codebase so I thought its something generally accepted so I think we are good to go.

However I see that you requested changes, is this related to Lightning ? Does Bisq-Easy support lightning invoices? I thought you mentioned in a previous comment that we should leave that for another PR. If the current implementation doesn't break anything then I think we should go as is and create a separate issue?

Let me know your thoughts!

Reply:
Yes we need at least only do the BTC checks if LN is not used. And yes we support LN now. Have you done some test trades?

Reply:
@HenrikJannsen I've done test trades with bitcoin only (bitcoin addresses and txIds)  - I can try do some lightning to see how it works in a few mins

Reply:
But I can do that as well in a follow up PR... Will merge so that it makes it into the release...